libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -210,7 +211,20 @@ def bind(self, **kwargs: Any) -> Runnable[Input, Output]:
         """
         Bind arguments to a Runnable, returning a new Runnable.
         """
-        return RunnableBinding(bound=self, kwargs=kwargs)
+        return RunnableBinding(bound=self, kwargs=kwargs, config={})
+
+    def with_config(
+        self,
+        config: Optional[RunnableConfig] = None,
+        # Sadly Unpack is not well supported by mypy so this will have to be untyped
+        **kwargs: Any,
```

Comment:
do we need to support kwargs at all? shouldn't everything just be passed in config, why do we need 2 ways to do same thing

Reply:
Because this is easier to use